utility.lua

```diff
@@ -1697,7 +1697,7 @@ function Auxiliary.RitualUltimateOperation(filter,level_function,greater_or_equa
 					tc:CompleteProcedure()
 				end
 				if extra_operation then
-					extra_operation(e,tp,eg,ep,ev,re,r,rp,tc)
+					extra_operation(e,tp,eg,ep,ev,re,r,rp,tc,mat)
```

Comment:
`mat` is a local var which is not in this scope

Reply:
https://github.com/Fluorohydride/ygopro-scripts/pull/1728/commits/27f4bf5fc722ad58e3d1ed542742106ba67b29a8 fixed.